aspnetcore/security/authentication/aad-ui/samples/WebAAD6/Areas/Identity/Pages/Account/SignOut.cshtml.cs

```diff
@@ -0,0 +1,20 @@
+using Microsoft.AspNetCore.Authentication.AzureAD.UI;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace WebAAD6.Areas.Identity.Pages.Account
+{
+    public class SignOutModel : PageModel
+    {
+        public void OnGet()
+        {
+        }
+        #region snippet
+        public void OnPost()
+        {
+            // SignOut is done in this order to ensure local log out
+            // even if the remote logout fails.
+            SignOut(AzureADDefaults.CookieScheme, AzureADDefaults.OpenIdScheme);
```

Comment:
warning CS0618: 'AzureADDefaults' is obsolete: 'This is obsolete and will be removed in a future version. Use Microsoft.Identity.Web instead. See https://aka.ms/ms-identity-web.'